powersimdata/scenario/move.py

```diff
@@ -1,4 +1,9 @@
+from fs.copy import copy_dir
+from fs.walk import Walker
+
+from powersimdata.data_access.data_access import get_ssh_fs
 from powersimdata.scenario.state import State
+from powersimdata.utility import server_setup
 
 
 class Move(State):
```

Comment:
`State` --> `Ready`?

Reply:
I guess the `print_scenario_info` method is unnecessary

Reply:
Good catch, done